README.md

```diff
@@ -1,24 +1,116 @@
-# README
+# 📗 Table of Contents
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-Things you may want to cover:
 
-* Ruby version
+# 📖 [Whispers] <a name="about-project"></a>
 
-* System dependencies
+**[Whispers]** is a Ruby on Rails blogs website that users can use to write their blogs, like a blog or comment on other users blogs.
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+### Tech Stack <a name="tech-stack"></a>
 
-* Database initialization
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://ruby.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* How to run the test suite
+### Key Features <a name="key-features"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+- **[Create a User]**
+- **[Add a Comment]**
+- **[Add a Like]**
 
-* Deployment instructions
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+In order to run this project you need:
+
+Ruby installed in your local machine
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd swift
+  git clone https://github.com/Solobarine/swift
+```
+
+### Install
+
+Install this project with:
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails s
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+👤 **Akpuru Solomon
+
+- GitHub: [@githubhandle](https://github.com/solobarine)
+- Twitter: [@twitterhandle](https://twitter.com/solomonakpuru)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/solomon-akpuru)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## FUTURE FEATURES <a name="future-features"></a>
+
+- JWT
+- Views
+- User Authentication
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] As both previous reviewers mentioned your issues page link is broken! Kindly fix it as suggested.


<img width="536" alt="Screenshot 2023-05-04 at 21 45 46" src="https://user-images.githubusercontent.com/69011963/236271382-9e82be24-44ce-4d61-86b8-f3f23a40ef31.png">